webapp/channels/src/components/widgets/popover/index.tsx

```diff
@@ -19,10 +19,9 @@ interface Props {
     style?: React.CSSProperties;
     onMouseOut?: React.MouseEventHandler<BSPopover>; // didn't find a better way to satisfy typing, so for now we have a slight 'bootstrap leakage'
     onMouseOver?: React.MouseEventHandler<BSPopover>;
-    ref?: React.Ref<BSPopover>;
 }
 
-const Popover = ({
```

Comment:
I am not a big fan of `forwardRef` (in theory passing the prop should do exactly the same) but at least firefox was complaining in the console, so I added it here.

Reply:
I think it's because there's nothing to get a reference to for function components unless you explicitly pass one, and there's probably a minor overhead to adding a ref or even checking for a prop named `ref`, so they don't want to do that automatically